.travis.yml

```diff
@@ -1,19 +1,25 @@
 language: php
-
+stages:
+  - test
+  - 
 php:
   - 5.6
   - 7.0
   - 7.1
   - 7.2
   - 7.3
-
 # faster builds on new travis setup not using sudo
 sudo: false
 
 install:
   - '[[ -z "$CI_USER_TOKEN" ]] || composer config github-oauth.github.com ${CI_USER_TOKEN};'
   - travis_retry composer self-update && composer --version
-  - travis_retry composer update --prefer-dist --no-interaction
-
+  - travis_retry composer install --prefer-dist --no-interaction
+  - phpenv config-rm xdebug.ini
+  - if [[ "$TRAVIS_PHP_VERSION" == "7.3" ]]; then pecl install -f pcov; fi
+  
 script:
   - php ./vendor/bin/codecept run
+  - if [[ "$TRAVIS_PHP_VERSION" == "7.3" ]]; then php ./vendor/bin/codecept run --coverage-xml; fi
```

Comment:
Tests are executed twice on 7.3, wouldn't it be better to use `!= "7.3"` condition in a previous line?

Reply:
Yeah, but tests are fast and I want it to be as simple as possible.